rskj-core/src/main/java/co/rsk/logfilter/BlocksBloom.java

```diff
@@ -67,12 +67,22 @@ public boolean hasBlockBloom(long blockNumber) {
     }
 
     public void addBlockBloom(long blockNumber, Bloom blockBloom) {
+        addBlockBloomInternal(blockNumber, blockBloom, false);
+    }
+
+    public void addBlockBloomBackwards(long blockNumber, Bloom blockBloom) {
```

Comment:
Not sure if it is ok for a bloom to have the option of going in both directions (forward and backward). Could have sense to create a BackwardsBlooksBloom?

Reply:
Good point, this could lead to misuse of the bloom if sometimes called forward and sometimes backward. Will address it.